index.tt

```diff
@@ -6,152 +6,251 @@
   </div>
 </div>
 
 <hr />
 
 <div class="row-fluid">
-  <div class="span6">
-    <h2>
-      <a href="/news-rss.xml">
-        <img src="/logo/rss.png"/>
-      </a>
-      News
-    </h2>
-    [% INSERT "latest-news.xhtml" %]
-    <p><a href="news.html" class="btn btn-mini">More »</a></p>
+  <div class="span4">
+    <h2>Reproducible</h2>
+    <p>
+      Nix builds packages in isolation from each other. This ensures that they
+      are reproducible and don't have undeclared dependencies, so <strong>if a
+        package works on one machine, it will also work on another</strong>.
+    </p>
   </div>
-  <div class="span6">
-
-    <h2>
-      <a href="/blogs.xml">
-        <img src="/logo/rss.png"/>
-      </a>
-      Blog Posts
-    </h2>
-    <ul class="nixos-blogs">
-      [% FOREACH b IN JSON.Escape.json_decode(IO.All.new('blogs.json').all).channel.item.first(5); IF b; %]
-        <li>
-          <div class="nixos-blog">
-            [%
-              USE title = String(HTML.escape(b.title.remove('^[^:]*:').trim));
-              title = title.replace('^[a-z]', title.copy.truncate(1).upper)
-              author = b.title.split(':').0.trim;
-            %]
-            [%
-            %]
-            <div class="nixos-blog-title">
-              <a [% HTML.attributes(href => b.link) %]>[% title %]</a>
-            </div>
-            <div class="nixos-blog-author-info">
-              <span class="nixos-blog-author">[% HTML.escape(author) %]</span>
-              <span class="muted"> wrote on [% HTML.escape(b.pubDate) %]</span>
-            </div>
-          </div>
-        </li>
-      [% END; END %]
-      <p><a href="https://planet.nixos.org/" class="btn btn-mini">More »</a></p>
-    </ul>
-
-    <h3>Twitter</h3>
+  <div class="span4">
+    <h2>Declarative</h2>
     <p>
-      <a class="twitter-timeline" data-chrome="noheader nofooter transparent" href="https://twitter.com/nixos_org" data-widget-id="433975692670152704">Tweets by @nixos_org</a>
-      <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
+      Nix makes it <strong>trivial to share development and build
+        environments</strong> for your projects, regardless of what programming
+      languages and tools you’re using.
     </p>
+  </div>
+  <div class="span4">
+    <h2>Reliable</h2>
+    <p>
+      Nix’s ensures that installing or upgrading one package <strong>cannot
+        break other packages</strong>. It allows you to <strong>roll back to
+        previous versions</strong>, and ensures that no package is in an
+      inconsistent state during an upgrade.
+    </p>
+  </div>
+</div>
+
+
+<hr />
 
+<div id="packages-search" class="row-fluid text-center">
+  <div class="span12">
+    <h1>Choose from Thousands of Packages</h1>
```

Comment:
I'm not happy with this title, but idea is to show that we have lots of package. This is a common question among newcomers: *"do you have all the packages that i use?"*.

I would welcome a better title, but I'm fine with this title since we can always improve it later.

Reply:
It's actually not that bad. OK for now.